marketplace-client/src/marketplace/src/app/checkout/checkout.component.html

```diff
@@ -0,0 +1,274 @@
+          </tr>
+
+          <tr>
+            <th scope="row">Surname</th>
+            <td class="text-center">{{ this.orderDetailsForm.controls['surname'].value }}</td>
+          </tr>
+
+          <tr>
+            <th scope="row">Phone</th>
+            <td class="text-center">{{ this.orderDetailsForm.controls['phone'].value }}</td>
+          </tr>
+
+          <tr>
+            <th scope="row">Address</th>
+            <td class="text-center">{{ this.orderDetailsForm.controls['address'].value }}</td>
+          </tr>
+
+          <tr>
+            <th scope="row">Delivery Time</th>
+            <td class="text-center">{{ this.orderDetailsForm.controls['deliveryTime'].value }}</td>
+          </tr>
+
+          <tr>
+            <th scope="row">Disturb</th>
+            <td class="text-center" *ngIf="this.orderDetailsForm.controls['disturb'].value">
+              Yes
+            </td>
+            <td class="text-center" *ngIf="!this.orderDetailsForm.controls['disturb'].value">
+              No
+            </td>
+          </tr>
+
+          <tr>
+            <th scope="row">Comment</th>
+            <td class="text-center">{{ this.orderDetailsForm.controls['comment'].value }}</td>
+          </tr>
+        </tbody>
+
+      </table>
+
+      <form (ngSubmit)="doOrder()">
+        <div class="row d-flex p-2 justify-content-end">
+          <div class="col-md-3">
+            <div class="form-group">
+              <button type="submit" class="btn btn-primary btn-block">Submit Order</button>
+            </div>
+          </div> 
+        </div>
+      </form>
+
+    
+  </div>
+
+  <div *ngIf="items.length !== 0">
+    <hr />
+    <h1 class="font-weight-bold">Products: {{getTotalPrice(items) | currency:'USD'}}</h1>
+    <hr />
+    
+    <div class="row">
+      <div class="col my-4">
+        <table class="table table-bordered">
+          <thead class="table-dark">
+            <tr class="text-center">
+              <th>Image</th>
+              <th>Title</th>
+              <th>Price</th>
+              <th>Quantity</th>
+              <th>Subtotal</th>
+            </tr>
+          </thead>
+          <tbody>
+
+            <tr *ngFor="let cartItem of this.items" class="text-center">
+              <td class="d-flex align-items-center justify-content-around">
+                <img
+                  src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fsmilytiffany%2F100x100-images%2F&psig=AOvVaw3xQvbQnb1NnoFIsFBlIvmJ&ust=1623328407610000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJjAmN3HivECFQAAAAAdAAAAABAJ"
+                  alt="{{cartItem.goods.goodName}}" height="60" class="order-1">
+              </td>
+
+              <td>
+                <p>{{cartItem.goods.goodName}}</p>
+              </td>
+
+              <td>
+                <p>{{getPrice(cartItem) | currency:'USD'}}</p>
+              </td>
+
+              <td>
+                <p>{{ getSubtotalPrice(cartItem) / getPrice(cartItem) }}</p>
```

Comment:
https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496